packages/core/src/Page/index.js

```diff
@@ -124,9 +125,13 @@ class Page {
   }
 
   prepareTemplateData(content, hasPageNav) {
-    const prefixedTitle = this.pageConfig.titlePrefix
-      ? this.pageConfig.titlePrefix + (this.title ? TITLE_PREFIX_SEPARATOR + this.title : '')
-      : this.title;
+    let { title } = this;
+    if (this.pageConfig.titlePrefix) {
+      title = this.pageConfig.titlePrefix + (title ? TITLE_PREFIX_SEPARATOR + title : '');
+    }
+    if (this.pageConfig.titleSuffix) {
+      title = (title ? title + TITLE_SUFFIX_SEPARATOR : '') + this.pageConfig.titleSuffix;
```

Comment:
LGTM.

When drafting my previous comment, I didn't really consider the edge case of prefix + separator + empty title + separator + suffix that we need to handle to ensure prefix + one separator + suffix, which complicates this seemingly simple logic here 😓. I think this is clearer (probably in the future we can refactor this, should we decide to allow custom separators). In hindsight, it would be great if we could write a unit test for this :)